stoplight.js

```diff
@@ -0,0 +1,25 @@
+$(document).ready(function(){
+var redButton=$("")
+var greenButton=$("")
+var yellowButton=$("")
+
+
+function stopFunction(){
+$("#red").toggleclss(.active#red)
```

Comment:
- Mispelled `.toggleClass`
- The argument passed into `.toggleClass` should be a string